addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -95,6 +97,15 @@ def _compute_available_operator_ids(self):
         for livechat_channel in self:
             livechat_channel.available_operator_ids = operators_by_livechat_channel[livechat_channel]
 
+    @api.constrains("review_link")
+    def _check_review_link(self):
+        for record in self.filtered("review_link"):
+            url = urlparse(record.review_link)
+            if not (url.scheme in ("http", "https") and url.netloc):
```

Comment:
:pick: I feel it's easier to read because we can just go left to right. With the current condition, we have to read the whole thing and then negate it.
```suggestion
            if url.scheme not in ("http", "https") or not url.netloc:
```